www/docs/en/dev/config_ref/index.md

```diff
@@ -476,7 +476,7 @@ This tag installs resource files into your platform, and is similar to the same
 
 Attributes(type) <br/> <span class="sub-header">Only for platform:</span> | Description
 ---------------- | ------------
-src(string) <br/> ==iOS== ==Android==| *Required* <br/> Location of the file relative to `plugin.xml`. If the src file can't be found, the CLI stops and reverses the installation, issues a notification about the problem, and exits with a non-zero code.
+src(string) <br/> ==iOS== ==Android==| *Required* <br/> Location of the file relative to `config.xml`. If the src file can't be found, the CLI stops and reverses the installation, issues a notification about the problem, and exits with a non-zero code.
```

Comment:
`the CLI stops and reverses the installation` is this still applicable to the non-plugin context? AFAIK this could be just a `prepare` or `build` with no installation involved.